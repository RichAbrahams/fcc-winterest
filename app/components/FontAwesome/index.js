/**
*
* FontAwesome
*
*/

import React from 'react';
import { FA } from './styles';


function FontAwesome({ name, size, color, transform }) {
  return (
    <FA
      name={name}
      size={size}
      color={color}
      transform={transform}
    />
  );
}

FontAwesome.propTypes = {
  name: React.PropTypes.string,
  size: React.PropTypes.string,
  color: React.PropTypes.string,
  transform: React.PropTypes.string,
};

export default FontAwesome;
