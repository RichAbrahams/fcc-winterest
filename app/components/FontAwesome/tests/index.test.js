import React from 'react';
import renderer from 'react-test-renderer';
import FontAwesome from '../index';

it('renders correctly', () => {
  const tree = renderer.create(
    <FontAwesome />).toJSON();
  expect(tree).toMatchSnapshot();
});
