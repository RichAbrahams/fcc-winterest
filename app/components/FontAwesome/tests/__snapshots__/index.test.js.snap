exports[`test renders correctly 1`] = `
<span
  aria-hidden={true}
  className="fa fa-undefined cRIVAo"
  color={undefined}
  transform={undefined} />
`;
